presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -305,13 +305,24 @@ Property Name                                        Description
                                                      running in EC2, or when the catalog is in a different region.
                                                      Example: ``us-east-1``
 
+``hive.metastore.glue.endpoint-url``                 Glue API endpoint URL (optional). Might be useful in
```

Comment:
Remove "Might be" and simplify .. so .. "Useful when all requests to Glue should be proxied.